pyserini/index/lucene.py

```diff
@@ -0,0 +1,31 @@
+#
+# Pyserini: Reproducible IR research with sparse and dense representations
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from jnius import autoclass
+import sys
+import os
+
+args = sys.argv[1:]
+
+# argument check
```

Comment:
I have a suggestion here: how about we go through the args and change double dashes back into single dashes. That way we can transparently fix the differences between Java and Python args.